### ISSUE TITLE
Needs docs

### ISSUE THREAD

**Issue by benlk** [INN/client-hosting-manager#7]
- [ ] What capabilities are managed?
- [ ] How are they managed?

**Comment by benlk** [comment]
https://github.com/INN/client-hosting-manager/blob/692438feaf5e9bf2b5b6a0fbdbb27d919d809fbf/client-hosting-manager.php#L190
```php
			'install_plugins',
			'activate_plugins',
			'delete_plugins',
			'edit_plugins',
			'install_themes',
			'switch_themes',
			'edit_themes',
			'update_core',
			'update_plugins',
			'update_themes',
```

**Comment by benlk** [comment]
https://wordpress.org/support/topic/what-is-the-point-of-this-plugin/

**Comment by rclations** [comment]
Re: that support case above, Mike and I talked at WordCamp for Publishers. He posted that review then came to talk to me afterwards. It seems he didn't pay attention to the warnings when setting up the plugin and just copy/pasted the example, then was upset when he locked himself out of his site.

We chatted for a bit about potential solutions and decided that the following would help mitigate his concerns:
1. If there is no user with an email address from the whitelisted domain, don't change anything.
2. Add a notice to the bottom of wp-admin or somethere else prominent indicating that permissions have been limited, and list the org contact email (from the plugin settings) so folks can get in touch if they think they should have more permissions.
